tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -261,6 +264,16 @@ public void Dispose()
             _waf?.Dispose();
         }
 
+        private void AsmDDProductConfigChanged(object sender, ProductConfigChangedEventArgs e)
+        {
+            var asmDD = e.GetDeserializedConfigurations<string>().FirstOrDefault();
```

Comment:
maybe this could be deserialized directly to `JToken[]` and passed to the waf as is, without a first step to string?